tf2onnx/onnx_opset/nn.py

```diff
@@ -59,8 +59,8 @@ def get_channels_last_permutation(spatial):
 
 def conv_convert_inputs(ctx, node, with_kernel=False, new_kernel_shape=None,
                         input_indices=None, output_indices=None, spatial=2):
-    """Convert input and kernel from tensorflow to onnx. This maybe require to
-        to insert transpose ops for input, kernel and output unless they are constants
+    """Convert input and kernel from tensorflow to onnx. This may be required to
```

Comment:
Thanks for catching the grammar issue,  please check the DCO requirement.

head to your local branch and run:
```
git commit --amend --no-edit --signoff
```
Now your commits will have your sign off. Next run
```
git push --force-with-lease origin mshr-h-patch-1
```